code/drasil-code/lib/Language/Drasil/Code/Imperative/Import.hs

```diff
@@ -471,10 +469,19 @@ vecVecNumBfunc Dot = error "convExpr DotProduct"
 numVecVecBfunc :: NVVBinOp -> (SValue r -> SValue r -> SValue r)
 numVecVecBfunc Scale = error "convExpr Scaling of Vectors"
 
+-- Maps a 'ESSBinOp' to it's corresponding GOOL binary function.
+elementSetSetBfunc :: ESSBinOp -> (SValue r -> SValue r -> SValue r)
+elementSetSetBfunc SAdd = error "convExpr Adding an Element to a Set"
+elementSetSetBfunc SRemove = error "convExpr Removing an Element to a Set"
+
+-- Maps a 'ESSBinOp' to it's corresponding GOOL binary function.
+elementSetBoolBfunc :: (SharedProg r) => ESBBinOp -> (SValue r -> SValue r -> SValue r)
+elementSetBoolBfunc SContains = contains
+
 -- medium hacks --
 
 -- | Converts a 'Mod' to GOOL.
-genModDef :: (OOProg r) => Mod -> GenState (OO.SFile r)
```

Comment:
I would want @B-rando1 to double-check this change.

Reply:
It looks like the imports were changed up here:
<img src="https://github.com/user-attachments/assets/0bb508d3-958f-47fa-a683-b3b58e7f2dfc" width="600px">
I'm not sure, it seems like in general we avoid importing everything like that.  It might be better to only import what we need.  Especially when dealing with `Drasil.GOOL` and `Drasil.GProc`, which export a lot of things with the same name.

Either way, I would say that `SFile` should be imported qualified, so that all of its occurrences are either `OO.SFile` or `Proc.SFile`.

Reply:
It looks like this has been resolved.